pkg/kubectl/cmd/get.go

```diff
@@ -262,10 +279,13 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 
 	for ix := range objs {
 		var mapping *meta.RESTMapping
+		var original runtime.Object
```

Comment:
How to print versioned objects? Whenever I try to print versioned objects, for example v1, I get 

```
error: unknown type &v1.Pod{TypeMeta:unversioned.TypeMeta{Kind:"Pod", APIVersion:"v1"},....
```
Complete error is here: http://fpaste.org/331443/67944711/